seolondon/settings/__init__.py

```diff
@@ -197,6 +200,10 @@
             'PORT': os.environ['RDS_PORT'],
         }
     }
+elif 'DATABASE_URL' in os.environ:
```

Comment:
Again it is for the test instance in heroku.  

Though it could be good for future dev setting/setup.  Last time I setup development env, I had to do a few postgres change directly after restoring the db locally.  With this setup, it could simplify the development setup if we ever need to restore db from production again